examples/rcbilling-demo/src/tests/main.test.ts

```diff
@@ -3,6 +3,8 @@ import { Locator } from "playwright";
 
 const _LOCAL_URL = "http://localhost:3001/";
 const CARD_SELECTOR = "div.card";
+const PACKAGE_SELECTOR = "button.package";
+const RC_PAYWALL_TEST_OFFERING_ID = "e2e_tests_web_paywalls";
```

Comment:
@tonidero do you think we should move this to an env variable? if so, can you help me with it? 

I don't think its strictly necessary since it's not something that exposes any sensible data, but I'll let you be the judge of that

Reply:
I think it's fine for the offering id to be hardcoded. Not really a secret. Could be worth moving to an ENV if we wanted to have different ones, but I don't think that's the case, so it's probably fine to leave here.